spanner_prober/prober/proberlib.go

```diff
@@ -78,8 +86,104 @@ var (
 		Aggregation: view.Count(),
 		TagKeys:     []tag.Key{resultTag, opNameTag},
 	}
+
+	poolSize = 2
```

Comment:
Why is this not a command line variable?

Reply:
Definitely will be. In a following PR.